util/init.js

```diff
@@ -42,10 +56,7 @@ before('reset metrics db', async function() {
 
 
 before('Initialize aerogear-js-sdk', async function() {
-  client.execute(config => {
-    const { init } = window.aerogear.agApp;
-    window.aerogear.app = init(config);
-  }, mobileServices);
+  client.execute(initJsSdk, mobileServices);
```

Comment:
Does it make sense to initialize the app before all tests even if not all tests are using it? For example, the push tests have to recreate it because some of the key used to generate the app are known only after creating the variant inside the server and this happens during the test (This comment maybe is a more general discussion and not related that mutch to this PR)